tests/components/aprs/test_device_tracker.py

```diff
@@ -302,6 +302,44 @@ def test_aprs_listener_rx_msg_no_position(mock_ais: MagicMock) -> None:
     see.assert_not_called()
 
 
+def test_aprs_listener_tx_msg_object(mock_ais: MagicMock) -> None:
+    """Test rx_msg with object."""
```

Comment:
Comment does not agree with function name.

Reply:
Thanks for catching that